carma-messenger-core/cpp_message/src/main.cpp

```diff
@@ -23,5 +24,7 @@ int main(int argc, char**argv)
 {
     ros::init(argc,argv,"cpp_message_node");
     cpp_message::Message node;
+    Message_cpp::MessageConsumer node_2;
```

Comment:
I think this will prevent `node.run()` from happening, as `node_2.run()` blocks while the node is running. These MobilityMessage callbacks/etc should probably just be integrated into the existing node to that it all runs as a single unit. We can discuss if you want.

Reply:
Changed this, geofence node now launches from inside the MessageConsumer run function.